src/views/AddItem.jsx

```diff
@@ -1,3 +1,84 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { React, useState } from 'react';
+
+import { addItem } from '../api/firebase';
+
+//we need listToken (listID) passing from App.js
+
+export function AddItem({ listToken }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(7);
+
+	const submitForm = (e) => {
+		e.preventDefault();
+		try {
+			addItem(listToken, { itemName, daysUntilNextPurchase });
+			alert('Item is saved in the database.');
```

Comment:
We should handle these error/success messages ourselves rather than use alert. You don't need to style it or anything at this point, but just adding a piece of state that you can display on the page and update in this eventHandler. This will give us a lot more flexibility down the road for being both stylish and accessible.